sound_play/scripts/soundplay_node.py

```diff
@@ -1,9 +1,9 @@
 #!/usr/bin/env python
-
+# -*- coding: utf-8 -*-
 #***********************************************************
 #* Software License Agreement (BSD License)
 #*
-#*  Copyright (c) 2009, Willow Garage, Inc.
+#*  Copyright (c) 2016 JSK Lab.
```

Comment:
Having made significant updates, there is nothing wrong with adding your own copyright line to the file, but you should *not* delete the old one.